blocks/footer/footer.js

```diff
@@ -13,7 +56,21 @@ export default async function decorate(block) {
   const resp = await fetch(`${footerPath}.plain.html`);
   const html = await resp.text();
   const footer = document.createElement('div');
+
   footer.innerHTML = html;
   await decorateIcons(footer);
   block.append(footer);
+  addCSSToLinkHeadings();
```

Comment:
for methods like this, you should pass the block in as a parameter, then base the selector queries on the block rather than the whole document

Reply:
Thanks Sean. Made the appropriate changes to the footer.js code to use block for querySelector methods rather than document.